quad_pyblish_module/plugins/tvpaint/create/create_playblast.py

```diff
@@ -54,7 +54,8 @@ def _create_new_instance(self):
             "variant": self.default_variant,
             "creator_attributes": {
                 "mark_for_review": self.mark_for_review,
-                "export_type": self.export_type
+                "export_type": self.export_type,
+                "self.extract_psd": self.extract_psd
```

Comment:
This should be:
```
"extract_psd": self.extract_psd
```

Reply:
You're right, i updated the code, thank you !